Assets/_Scripts/AIScripts/AISight.cs

```diff
@@ -67,62 +68,66 @@ private void FixedUpdate()
     }
     void OnTriggerStay(Collider other)
     {
-        bool hiding = Player.IsHiding;
-        behindWall = false;
-
-        //check if the player is in sight
-        if (GameObject.ReferenceEquals(other.gameObject, player[0]))
+        if (!stunned)
         {
-            //--------------------------------------------------------------------------------------------------------------------------------------------
-            //playerInSight = -1;//default false
-            //--------------------------------------------------------------------------------------------------------------------------------------------
-            Vector3 directionPlayer = other.transform.position - transform.position; //make a vector pointing at the player
-            float angle = Vector3.Angle(directionPlayer, transform.forward);//find angle between forward self and the player
+            bool hiding = Player.IsHiding;
+            behindWall = false;
 
-            if (angle < 0.5f * view)//if player is less than half of our view angle...
+            //check if the player is in sight
+            if (GameObject.ReferenceEquals(other.gameObject, player[0]))
```

Comment:
Yo wait, is this the better version of doing

if(other.gameObject == player[0])

cuz if so I had no idea this was a thing and I should have been using this

Reply:
Oh, this wasn't meant to be a review